versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/AbstractVersionsUpdaterMojo.java

```diff
@@ -205,10 +205,11 @@
     @Parameter( property = "changeRecorderOutputFile",
                 defaultValue = "${project.build.directory}/versions-changes.xml" )
     private File changeRecorderOutputFile;
+
     /**
      * The change recorder implementation.
      */
-    private ChangeRecorder changeRecorder;
+    private volatile ChangeRecorder changeRecorder;
```

Comment:
Is it accessed from multi thread somehow?

Reply:
Not needed indeed. Removed.